Formula/nqp.rb

```diff
@@ -0,0 +1,23 @@
+class Nqp < Formula
+  desc "Lightweight Perl 6-like environment for virtual machines"
+  homepage "https://github.com/perl6/nqp"
+  url "https://rakudo.perl6.org/downloads/nqp/nqp-2018.12.tar.gz"
+  sha256 "219db519ad5c1848e4528a56a506dd74b0839ca1d910788411f3bfedf5045d36"
+  head "https://github.com/perl6/nqp.git"
+
+  depends_on "moarvm"
+
+  def install
+    system "perl", "Configure.pl",
+      "--backends=moar",
```

Comment:
Tiny detail, but: please indent the continuation lines so `"--backends=moar",` is below `"Configure.pl",` (Homebrew style).